README.md

```diff
@@ -240,13 +240,12 @@ func main() {
 	dataCh := consumer.StartFetch()
 	fetchRes := make([]hstream.RecordId, 0, 100)
 	for res := range dataCh {
-		receivedRecords, err := res.GetResult()
-		if err != nil {
-			log.Printf("Fetch error: %s\n", err)
+		if res.Err != nil {
+			log.Printf("Fetch error: %s\n", res.Err)
 			continue
 		}
 
-		for _, record := range receivedRecords {
+		for _, record := range res.Result {
 			rid := record.GetRecordId()
 			log.Printf("receive recordId: %s\n", rid.String())
 			fetchRes = append(fetchRes, rid)
```

Comment:
should send `Ack` for each record, See https://github.com/hstreamdb/hstreamdb-go/blob/7b3daf1440acb7b7f3eb5ca4512f6831cebca1e7/integraion_test/subscription_test.go#L137-L143

Reply:
thx, fixed